releasenotes/notes/parallel-closeness-centrality-5165f62e370181eb.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    The :func:`~.closeness_centrality` method is now parallel for better performance.
+    The parallelism is toggled via the new `parallel_threshold` optional parameter.
+other:
```

I'd put this in the upgrade section because that's where we typically put breaking api changes. It's something the user may need to update when they upgrade to the next release.

```suggestion
upgrade:
```

---

Done.